Services/Pagination.php

```diff
@@ -0,0 +1,355 @@
+<?php
+
+namespace Azine\EmailBundle\Services;
```

Comment:
Actually, this is my coworker`s bundle and i took this with his permission, but of course it would be much easier for  me to use  KnpPaginatorBundle. I just followed this one (https://symfony.com/doc/2.7/bundles/best_practices.html#vendors) not to use 3d party libraries for reusable bundles.  But as i said i will gladly use KnpPaginatorBundle.

Reply:
https://symfony.com/doc/2.7/bundles/best_practices.html#vendors
this means you should not include for example `swiftmailer`-code anywhere in the folders of the email-bundle, but you should `require` it via composer.json